spec/features/user_can_login_account_spec.rb

```diff
@@ -29,4 +29,36 @@
     expect(page).to_not have_content "Login"
     expect(page).to have_content "Logout"
   end
+
+  scenario "I get redirected if my credentials are not correct" do
+    User.create(
+                first_name: "Jordan",
+                last_name: "Lawler",
+                username: "jlawlz",
+                password: "password"
+                )
+
+    visit '/login'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.